solarforecastarbiter/reference_forecasts/models.py

```diff
@@ -270,14 +270,19 @@ def gfs_quarter_deg_to_hourly_mean(latitude, longitude, elevation,
     output. GHI from NWP model cloud cover. DNI, DHI computed.
     Max forecast horizon 384 hours.
     """
-    cloud_cover, air_temperature, wind_speed = load_forecast(
```

Comment:
@alorenzo175 what do you think about this strategy of expanding the load window and then slicing later? I don't love it but don't know of a better way.

Reply:
I guess you had a bunch of slicing already so :man_shrugging: . But you should probably only slice the 1h unmixed since slie_args uses iloc. Maybe also check if start > end_1h and avoid 1h slicing and dicing when you can

Reply:
> But you should probably only slice the 1h unmixed

`load_forecast` function is going to return data outside of the `start:end` bounds for the other variables too. So I am confused about the recommendation.

> since slie_args uses iloc

whoops, forgot that `slice_args` uses `iloc` instead of `loc`. That will need to be changed. It doesn't look like the function is used anywhere.

> Maybe also check if start > end_1h and avoid 1h slicing and dicing when you can

The current changes to that code are incidental, but that's reasonable improvement in any case.

Reply:
> load_forecast function is going to return data outside of the start:end bounds for the other variables too. So I am confused about the recommendation.

True, I was only thinking about cloud cover.